recipes/mTEDx/LM/RNNLM/hparams/train.yaml

```diff
@@ -0,0 +1,109 @@
+# ############################################################################
+# Model: RNNLM of E2E ASR
+# Tokens: unigram
+# losses: NLL
+# Training: mTEDx
+# Authors:  Ju-Chieh Chou 2020, Jianyuan Zhong 2021, Mohamed Anwar 2022
+# ############################################################################
+
+# Seed needs to be set at top of yaml, before objects with parameters are made
+seed: 1234
+__set_seed: !apply:torch.manual_seed [!ref <seed>]
+output_folder: !PLACEHOLDER
+save_folder: !ref <output_folder>/save
+train_log: !ref <output_folder>/train_log.txt
+
+# Data files
+# The data_folder is needed because we train the LM on the training
+# transcriptions of original mTEDx as well.
+data_folder: !PLACEHOLDER
+langs:
+   - !PLACEHOLDER
```

Comment:
Can this be addressed via command line override ?
( @pplantinga  I hope so - but how ? )

Reply:
it should have been:
```
langs:
    - fr
```
Just forgot to change it :)